src/o365/graph/commands/teams/teams-user-list.ts

```diff
@@ -0,0 +1,146 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import Utils from '../../../../Utils';
+import { GroupUser } from '../o365group/GroupUser';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  role?: string;
+}
+
+
+class TeamsUserListCommand extends GraphItemsListCommand<GroupUser> {
```

Comment:
Following the naming convention, the class should be named `GraphTeamsUserListCommand`